### PR TITLE
feat: add support for the new endpoint to sync process port assigment with its functionality graph

### LINKED ISSUE #111: add support for the new endpoint to update ports on a functionality graph
Blocked by:

- [x] https://github.com/ospin-web-dev/hambda/issues/1734

### Deliverables
- [ ] add the API call to the new endpoint to update the ports

### LINKED ISSUE #112: add Support for the new endpoint to sync process port assigment with its functionality graph

### PR DESCRIPTION
**Pull request by FelixMZ2018**
feat: added sync process port
feat: added update ports on device

### CODE REVIEW

File: src/device/functionalityGraph/ports/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const update = require('./update')
+
```
Comment (FelixMZ2018) [inline src/device/functionalityGraph/ports/index.js:2]:
```suggestion

/**
 * @namespace nexus.device.functionalityGraph.ports
 */
```

File: src/process/physicalMapping/index.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const update = require('./update')
+
```
Comment (FelixMZ2018) [inline src/process/physicalMapping/index.js:2]:
```suggestion

/**
 * @namespace nexus.process.physicalMapping
 */
```

File: test/src/device/functionalityGraph/ports/update.test.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    expect(API.patch)
+      .toHaveBeenCalledWith(
+        'device',
+        `${params.deviceId}/functionality-graphs/${params.fctGraphId}/ports`,
+        { body: params.body, ...DEFAULT_REQ_OPTS },
+      )
+  })
+
+  testDefaultHTTPResponses(updatePorts, 'patch', [ params ])
+})
```
Comment (vdiezel) [inline test/src/device/functionalityGraph/ports/update.test.js]: We have an even shorter abstraction: the `testHTTPEndpoint` test helper. Applies to both tests.

**Review by vdiezel — CHANGES_REQUESTED** [review]
Solid. 
Just for the test suites we have a helper function that reduces the boilerplate even further.

**Review by vdiezel — CHANGES_REQUESTED** [review]
The test files can be cleaned up even further :)

check the licence add test.

**Review by vdiezel — APPROVED** [review]

**Pull request merged**